app/io/pathfinder/websockets/controllers/VehicleSocketController.scala

```diff
@@ -1,9 +1,29 @@
 package io.pathfinder.websockets.controllers
 
-import io.pathfinder.models.Vehicle
-import io.pathfinder.websockets.ModelTypes
-
+import io.pathfinder.models.{Commodity, Cluster, Vehicle}
+import io.pathfinder.routing.Action.{DropOff, PickUp}
+import io.pathfinder.routing.{Action, Route}
+import io.pathfinder.websockets.{WebSocketMessage, ModelTypes}
+import io.pathfinder.websockets.WebSocketMessage.{Route => RouteMsg, Routed}
+import scala.collection.JavaConversions.asScalaBuffer
 /**
  * manages vehicle API calls
  */
-object VehicleSocketController extends WebSocketCrudController[Vehicle](ModelTypes.Vehicle,Vehicle.Dao)
+object VehicleSocketController extends WebSocketCrudController[Vehicle](ModelTypes.Vehicle,Vehicle.Dao) {
```

Comment:
I guess you could. I made the modeltypes mostly for validating/parsing and multiplexing requests. I didn't consider the daos at the time I made it.